### ISSUE TITLE
`explicit-function-return-type` and JavaScript files

### ISSUE THREAD

**Issue by Conaclos** [denoland/deno_lint#1055]
Hi!

### Lint Name
[explicit-function-return-type](https://lint.deno.land/?q=&all=on#explicit-function-return-type)

### Code Snippet

```js
export function f() {}
```

```json
{
  "lint": {
    "files": {
      "include": ["src/index.js"]
    },
    "rules": {
      "tags": ["recommended"],
      "include": ["explicit-function-return-type"]
    }
  }
}
```

### Expected Result

The rule should not be applied on JavaScript files.

### Actual Result

```
export function f() {}
       ^^^^^^^^^^^^^^^
    src/index.js:1:7

    hint: Add a return type to the function signature
    help: for further information visit https://lint.deno.land/#explicit-function-return-type
```

### Version

deno 1.23.1 (release, x86_64-unknown-linux-gnu)
v8 10.4.132.8
typescript 4.7.2

**Comment by heygarrett** [comment]
This appears to still be an issue for `.mjs` files.
